src/database/DataTypes/StocktakeItem.js

```diff
@@ -98,16 +98,6 @@ export class StocktakeItem extends Realm.Object {
       }
 
       let difference = quantity - this.countedTotalQuantity;
-      // In case number is entered in Actual Quantity field, but it's the same as
-      // snapshot quantity, we want to remove 'Not Counted' placeholder
-      if (difference === 0) {
-        this.batches.forEach(stocktakeBatch => {
-          stocktakeBatch.countedTotalQuantity = stocktakeBatch.snapshotTotalQuantity;
-          database.save('StocktakeBatch', stocktakeBatch);
-        });
-        return;
-      }
-      // Actual Quantity is not snaphot quantity
```

Comment:
I've tested this, the app correctly shows "Not Counted" when no value was set, not if the value is the same as snapshot.

If snapshot was 30, you counted 25. Then selected that field and entered 25 again, it'd see difference 0 and set it to 30.

Reply:
If there is more then one batch, and actual quantity is set to snapshot quantity we get this:
<img width="324" alt="screen shot 2018-03-19 at 8 37 18 pm" src="https://user-images.githubusercontent.com/26194949/37602975-a6ee9cf8-2bb5-11e8-876f-2a338fb43a31.png">

Reply:
Ah I see! Will fix.